src/lib/containers/UserCardSmall.tsx

```diff
@@ -96,64 +96,45 @@ export const UserCardSmall = (props: UserCardSmallProps) => {
     <></>
   )
 
-  const fullName = showFullName ? (
-    <span className={'user-fullname'}>
-      &nbsp;
-      {`${userProfile.firstName ?? ''}`}&nbsp;{`${userProfile.lastName ?? ''}`}
-    </span>
-  ) : (
-    <></>
-  )
+  const fullName =
+    showFullName && (userProfile.firstName || userProfile.lastName) ? (
+      <span className={'user-fullname'}>
+        {`${userProfile.firstName ?? ''}`}
+        {userProfile.firstName && userProfile.lastName ? '\u00A0' : ''}
+        {`${userProfile.lastName ?? ''}`}
+      </span>
+    ) : null
+
+  const Tag = showCardOnHover || !disableLink ? 'a' : 'span'
 
-  return showCardOnHover ? (
+  const style: React.CSSProperties = showCardOnHover
```

Comment:
Should probably avoid nested ternary operators for readability.  This case is pretty straightforward, but could be expanded into an if/then conditional instead.